### PR TITLE
Handle CentOS vm repo

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -161,7 +161,8 @@ install-sysvinit: install-init
 	install network/qubes-iptables $(DESTDIR)/etc/init.d/
 
 install-rh: install-systemd install-systemd-dropins install-sysvinit
-	install -D -m 0644 misc/qubes-r4.repo $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
+	install -D -m 0644 misc/qubes-r4.repo.in $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
+	sed -i "s/@DIST@/$(shell echo $(DIST) | sed 's/[[:digit:]]//g')/g" $(DESTDIR)/etc/yum.repos.d/qubes-r4.repo
```
Comment (marmarek) [inline Makefile]: what about `${DIST%%[0-9]*}` instead of `$(shell ...)`?

Comment (fepitre) [reply Makefile]: Oh I didn't know such syntax in Makefile, in Bash I would have used ${DIST//[[:digit:]]}. So I tried but got empty result. Will check.

Comment (marmarek) [reply Makefile]: Yes, I meant shell syntax for that. You may need to use `$$` instead of `$`.

**Comment by marmarek** [comment]
Ok, so leave as is (but squash into the first commit).

**Comment by fepitre** [comment]
Done.

**Pull request merged**